typescript-loops/typescript-loops-quiz-notes.md

```diff
@@ -0,0 +1,54 @@
+# typescript-loops-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What is the purpose of a loop?
+  //The reuse the code (loops allow you to execute a block of code multiple times), working with collections (you can iterate over each item in the collection and perform an operation on it), performance (it can be more efficient that repeating code), control flow (loops with conditional statements allow control of the flow of the program)
+- What is the purpose of a **condition** expression in a loop?
+  //the code will execute while the condition expression is true, it is used to indicate when the loop should terminate
+- What does "iteration" mean in the context of loops?
+  //iteration in this context means that the code block will run
+- _When_ does the **condition** expression of a `while` loop get evaluated?
+  //when the assigned variable keeping track of the number of iterations the loop executes and comparing that to the amount of iterations we want, so before each iteration
```

Comment:
Remember in while loop we don't know how many iterations we want... The condition expr of while loop is evaluated before each iteration of the loop...